src/main/java/org/kairosdb/datastore/cassandra/CassandraDatastore.java

```diff
@@ -237,6 +246,9 @@ public void putDataPoint(String metricName,
                 //Check tag names and values to write them out
                 for (final String tagName : tags.keySet()) {
                     if (!tagNameCache.isKnown(tagName)) {
+                        if ( span != null) {
+                            span.setTag("tag_name_not_in_cache", Boolean.TRUE);
```

Comment:
Are these tag names used more than once?  We should probably set them as constants somewhere for easy reference, even if not.

Reply:
These tag names are used only once.